### DISCUSSION TITLE
Can you add a multi-documnet chat option?

### DISCUSSION THREAD

**Discussion by Christoway** [arc53/DocsGPT#311] [Ideas]
Hello,

Can you add a multiple document chat option in the future? I'd like to set up on my computer and link it up to a folder full of document for it to work with.

**Comment by dartpain** [comment]
Currently you can upload a .zip file with multimple files, but I agree we should support folders in the frontend

**Comment by Christoway** [comment]
Sounds good. I do document analysis and have been searching for an open-source solution to try out.

**Comment by gl2007** [comment]
> Sounds good. I do document analysis and have been searching for an open-source solution to try out.

Did you find anything so far?
